README.rst

```diff
@@ -1,78 +1,113 @@
+============================== =========================================================================================
+
+Once the redirect URI is updated, the app is ready to be used from Curity.
+
+Creating an Amazon Cognito Authenticator in Curity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The easiest way to configure a new Amazon Cognito authenticator is using the Curity admin UI. The configuration for this can be downloaded as XML or CLI commands later, so only the steps to do this in the GUI will be described.
+
+1. Go to the ``Authenticators`` page of the authentication profile wherein the authenticator instance should be created.
+2. Click the ``New Authenticator`` button.
+3. Enter a name (e.g., ``cognito1``).
+4. For the type, pick the ``Aws`` option:
+
+    .. figure:: docs/images/authenticator-type-in-curity.png
+        :align: center
+        :width: 600px
+
+5. On the next page, you can define all of the standard authenticator configuration options like any previous authenticator that should run, the resulting ACR, transformers that should executed, etc. At the bottom of the configuration page, the Amazon Cognito-specific options can be found.
+
+    .. note::
+
+        The Amazon Cognito-specific configuration is generated dynamically based on the `configuration model defined in the Java interface <https://github.com/curityio/aws-authenticator/blob/master/src/main/java/io/curity/identityserver/plugin/aws/config/AWSAuthenticatorPluginConfig.java>`_.
+
+6. Certain required and optional configuration settings may be provided. One of these is the ``HTTP Client`` setting. This is the HTTP client that will be used to communicate with the Amazon Cognito OAuth server's token and user info endpoints. This will only be required if the calls to Amazon Cognito are made through a forwarding proxy or there is an benign SSL Man-in-the-Middle that uses some untrusted SSL certificate. To define this, do the following:
+
+    A. click the ``Facilities`` button at the top-right of the screen.
+    B. Next to ``HTTP``, click ``New``.
+    C. Enter some name (e.g., ``awsClient``).
+    D. Toggle on the ``Use Truststore`` or ``Proxy`` options as needed. When finished, click ``Apply``.
+
+    .. figure:: docs/images/create-http-client.png
+        :align: center
+        :width: 600px
+
+
+7. Back in the Amazon Cognito authenticator instance that you started to define, select the new HTTP client from the dropdown if you created one; otherwise, leave this setting blank.
 
-Then, build the plugin by: ``mvn clean package``
+    .. figure:: docs/images/select-http-client.png
+        :align: center
+        :width: 400px
 
-Install plugin
-~~~~~~~~~~~~~~
+8. In the ``Client ID`` textfield, enter the client ID from the Amazon Cognito app configuration. This is the auto-generated ID that was shown after you register it. Also enter the matching ``Client Secret`` and ``Domain``(Domain is the URL to your Cognito App).
```

Comment:
need a space between 
```
Domain``
``` and `(Domain`. Otherwise, it's not rendered properly as mono-spaced text.

Reply:
ok